### PR TITLE
permissions set in AuthenticationResponse

### PR DESCRIPTION
**Pull request by AniJan**

### CODE REVIEW

File: identity-api/identity-api-facade/src/main/java/com/sflpro/identity/api/endpoints/AuthenticationEndpoint.java [hunk 1]
```diff
@@ -68,6 +72,14 @@
         //Try to authenticate
         try {
             AuthenticationResponse authResponse = authService.authenticate(authRequest);
+            final List<Role> roles = authResponse.getIdentity().getRoles();
+            final Set<String> permissions = new HashSet<>();
+            roles.forEach(role -> {
```
Comment (sfl-sonar) [inline identity-api/identity-api-facade/src/main/java/com/sflpro/identity/api/endpoints/AuthenticationEndpoint.java]: MINOR Remove useless curly braces around statement [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)

File: identity-api/identity-api-facade/src/main/java/com/sflpro/identity/api/endpoints/AuthenticationEndpoint.java [hunk 2]
```diff
@@ -68,6 +72,14 @@
...
+                role.getPermissions().forEach(permission -> {
```
Comment (sfl-sonar) [inline identity-api/identity-api-facade/src/main/java/com/sflpro/identity/api/endpoints/AuthenticationEndpoint.java]: MINOR Remove useless curly braces around statement [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)

**Comment by sfl-sonar** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by dnavre** [comment]
This resolves #28

**Pull request merged**